opencog/guile/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ TARGET_LINK_LIBRARIES(logger
 	smob
 	${GUILE_LIBRARIES}
 	${COGUTIL_LIBRARY}
+    ruleengine # for the ure logger
```

Comment:
Oof. This sure smells like a circular dependency, to me.  I think it would be a lot better if this got loaded in the scheme module, rather than being linked in the C code.